compiler/testData/diagnostics/tests/controlStructures/catchInnerClassesOfGenerics.fir.kt

```diff
@@ -3,39 +3,39 @@
 // JAVAC_EXPECTED_FILE
 
 class OuterGeneric<T> {
-    inner class ErrorInnerExn : Exception()
+    <!INNER_CLASS_OF_GENERIC_THROWABLE_SUBCLASS!>inner class ErrorInnerExn : Exception()<!>
```

Comment:
Diagnostic range is too wide. Please compare with `*.kt` file. You should use positioning strategy here. See e.g. `SourceElementPositioningStrategies.DECLARATION_NAME` and its usages, and the definition of `Errors.INNER_CLASS_OF_...`.

Reply:
Normally you should get `FIR_IDENTICAL` for this file (full identity to *.kt file)